packages/react/src/components/Tag/Tag.tsx

```diff
@@ -99,7 +106,7 @@ const Tag = <T extends React.ElementType>({
   className,
   id,
   type,
-  filter,
+  filter, // remove filter in next major release
```

Comment:
```suggestion
  filter, // remove filter in next major release - V12
```

Might be good to tag it with `V12` so it is easier to find these comments when we start the V12 effort